content/docs/quickstart/symphony.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: Run a Symphony app
+title: Run a Symfony app
```

Comment:
Good catch! Should we rename the file as well?

Reply:
Yes, I find that best practice.

Changed in following commit: https://github.com/neondatabase/website/pull/59/commits/6677f83ffb8c32986935a20a789b8eaeb4f9a6d1